src/encoded/static/components/cart/cart.js

```diff
@@ -1671,3 +1694,341 @@ Cart.contextTypes = {
 };
 
 export default Cart;
+
+
+/**
+ * Allows the user to select a single assembly for the genome browser.
+ */
+const CartAssemblySelector = ({ selectedAssembly, assemblies, onChangeAssembly }) => {
+    /**
+     * Called when the user makes an assembly selection.
+     * @param {object} event Synthetic even for user's selection
+     */
+    const onChange = (event) => {
+        onChangeAssembly(event.target.value);
+    };
```

Comment:
Is there a reason you have this as a separate function? Couldn't you have something like the following on line 1713?

> onChange={(e) => onChangeAssembly(e.target.value)}

Reply:
Purely inertia. After years, it still doesn’t occur to me.